src/app/header/header.component.ts

```diff
@@ -3,12 +3,13 @@ import { Component, HostListener, Inject, OnDestroy, OnInit } from '@angular/cor
 import { ActivatedRoute } from '@angular/router';
 import gql from 'graphql-tag';
 import * as qs from 'qs';
-import { Observable, Subject, Subscription } from 'rxjs';
-import { map, throttleTime } from 'rxjs/operators';
+import { Observable, of, Subject, Subscription } from 'rxjs';
```

Comment:
Would recommend to change these imports to submodules, so github stops nitpicking about blacklisted imports.

`Importing this module is blacklisted. Try importing a submodule instead. `